gourmet/tests/test_ica_se_plugin.py

```diff
@@ -42,7 +42,7 @@ def test_url(self):
     def test_parse(self):
         # Setup
         parser = self.plugin.get_importer(DummyImporter)()
-        parser.soup = BeautifulSoup(self.text, convertEntities=BeautifulStoneSoup.XHTML_ENTITIES,)
+        parser.soup = BeautifulSoup(self.text, "xml")
```

Comment:
I think this brings the external dependency `lxml`. Do we need to add that somewhere?

Reply:
https://www.crummy.com/software/BeautifulSoup/bs4/doc/#installing-a-parser

Reply:
I think we could even use `lxml` here as we have a HTML web-page - it seems like `lxml` is the recommended parser anyway.

Reply:
According to the documentation, even putting `xml` introduces the dependency. I don't see that we have that in any dependency file. So if `lxml` is a dependency of gourmet, we have to put it somewhere.

Reply:
It seems like https://github.com/kirienko/gourmet/blob/master/requirements.txt already has the `lxml` package inside.

Reply:
Okay, then just the search was broken: https://github.com/kirienko/gourmet/search?q=lxml&unscoped_q=lxml shows no results for me. Thanks for checking.